tests/test_trigger_bisect_jobs.py

```diff
@@ -7,7 +7,6 @@
 import importlib.util
 import json
 import os.path
-import sys
 import re
```

Comment:
this is not related to the style I guess. Maybe a separate commit?

Reply:
It is related to style changes by removing unused changes. I only applied changes suggested by ruff

Reply:
Technically @b10n1k is correct. This is *not* a style change. I assume ruff is not just about coding style and complains about all sorts of "problems".